c2cgeoportal/scaffolds/create/apache/application.wsgi.mako

```diff
@@ -1,15 +1,28 @@
 import site
 import sys
 import re
+import os
+from logging.config import fileConfig
 
+% if docker == 'TRUE':
+root = "/app"
+% else:
 site.addsitedir("${python_path}")
+root = "${directory}"
+% endif
+
 
 # Remove site packages
 regex = re.compile("^/usr/lib/python.\../dist-packages$")
 sys.path = [p for p in sys.path if regex.match(p) is None]
 
-from pyramid.paster import get_app, setup_logging
+from pyramid.paster import get_app
```

Comment:
Apart from the fact that it was there before... I don't know. I've tried to move it up and I've seen no ill effect.
You want me to move it at the top?